scene/gui/control.cpp

```diff
@@ -2063,14 +2063,10 @@ Control *Control::find_next_valid_focus() const {
 		// If the focus property is manually overwritten, attempt to use it.
 
 		if (!data.focus_next.is_empty()) {
-			Node *n = get_node(data.focus_next);
-			Control *c;
-			if (n) {
-				c = Object::cast_to<Control>(n);
-				ERR_FAIL_COND_V_MSG(!c, nullptr, "Next focus node is not a control: " + n->get_name() + ".");
-			} else {
-				return nullptr;
-			}
+			Node *n = get_node_or_null(data.focus_next);
+			ERR_FAIL_NULL_V_MSG(n, nullptr, "Next focus node path is invalid: " + data.focus_next + ".");
```

Comment:
Should probably wrap all paths in single quotes in error messages, so it's clear when it's an empty string? But I don't quite remember if we do it this way consistently.

Reply:
Will do for this